pkg/debug/types.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package debug
 
 import (
-	"math/rand"
+	"crypto/rand"
```

Comment:
The math/random returns a new pseudo-random source seeded with the given value, and crypto/rand does not depend on a seed value.

So corresponding usage below in function needs to be changed as well.